### ISSUE TITLE
Phone number limitation of inputed digits

### ISSUE THREAD

**Issue by Iheanacho2027** [SORMAS-Foundation/SORMAS-Project#1209]
The number of digits for Nigerian phone numbers is 11, there should be a constraint that prevents users from inputing more than 11 digits and less than 11 digits to prevent errors when inputing phone numbers @chinedar @foyiri

**Comment by foyiri** [comment]
yes this will help provide accurate phone number for contact tracing
